stream-chat-android-client/build.gradle

```diff
@@ -19,12 +19,11 @@ ext {
 apply from: "${rootProject.projectDir}/scripts/publish-mavencentral.gradle"
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion Configuration.compileSdkVersion
     buildToolsVersion "29.0.3"
```

Comment:
Can also be extracted to a variable

Reply:
Oops, left these here accidentally. They can just be removed, the plugin sets these implicitly anyway.